aqandu/utils.py

```diff
@@ -21,34 +21,37 @@ def validateDate(dateString):
 
 def parseDateString(datetime_string):
     """Parse date string into a datetime object"""
-    return datetime.strptime(datetime_string, DATETIME_FORMAT).astimezone(timezone('US/Mountain'))
-
-
-def datetimeToBigQueryTimestamp(date):
-    return date.strftime(BQ_DATETIME_FORMAT)
```

Comment:
I did not put this in there.  This is legacy code from Oleks.  I suspect his understanding evolved and it might not be needed.  However, I was not sure, so I left it in.

Reply:
As long as the database query properly returns values with respect to the time zone it is unneeded. Previously, "America/Denver" was added to datetime params for BigQuery to accomplish this.